platform/net/ubuntu_net_manager.go

```diff
@@ -359,15 +359,17 @@ auto lo
 iface lo inet loopback
 {{ range .DHCPConfigs }}
 auto {{ .Name }}
-iface {{ .Name }} inet dhcp
+iface {{ .Name }} inet dhcp{{ range .PostUpRoutes }}
+post-up route add -net {{ .Destination }} netmask {{ .Netmask }} gw {{ .Gateway }}{{ end }}
 {{ end }}{{ range .StaticConfigs }}
 auto {{ .Name }}
 iface {{ .Name }} inet{{ .Version6 }} static
     address {{ .Address }}{{ if not .IsVersion6 }}
     network {{ .Network }}{{ end }}
     netmask {{ .NetmaskOrLen }}{{ if .IsDefaultForGateway }}{{ if not .IsVersion6 }}
     broadcast {{ .Broadcast }}{{ end }}
-    gateway {{ .Gateway }}{{ end }}
+    gateway {{ .Gateway }}{{ end }}{{ range .PostUpRoutes }}
+    post-up route add -net {{ .Destination }} netmask {{ .Netmask }} gw {{ .Gateway }}{{ end }}
```

Comment:
According to the [docs](http://manpages.ubuntu.com/manpages/artful/man8/route.8.html), it looks like when you're adding an IPv6 address with the `route` utility, you need to specify the address family to override the IPv4 default like so:

```
# route -A inet6 add <TARGET> gw <GW>
route -A inet6 add fe80::800:27ff:fe01:0/128 gw ::1
```

Can you please add logic/tests for determining the family of the IP address and using the correct route command?

Reply:
@mikexuu Yes. Only support ipv4 currently. I will add it.